### ISSUE TITLE
Add static code checks

### ISSUE THREAD

**Issue by twolodzko** [julia-actions/julia-runtest#32]
It would be great if static code checks were included. Currently this probably is blocked by not having easy access to the API of static linter: https://github.com/julia-vscode/StaticLint.jl/issues/14

**Comment by christopher-dG** [comment]
I doubt that such a feature would end up here, it's generally out of scope.

**Comment by SaschaMann** [comment]
I agree. Linting should rather be its own action. Happy to add you to the org if you want to create such an action and have it live in julia-actions.
